package.json

```diff
@@ -23,15 +20,17 @@
     "lint": "npm run lint:js",
     "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern **/**.min.js",
     "lint:js": "npm run lint:eslint -- . ",
-    "lint:staged": "lint-staged",
+    "prebuild": "npm run build:clean && npm run test",
+    "prepare": "npm run build",
```

Comment:
I'll leave it here to make sure the decision to switch prepublish->prepare is deliberate.
`prepare` runs before publish but also when you install from a local copy or from git. `prepublishOnly` does what it says ;)  - only runs when creating a tarball to publish to npm.

Reply:
Right. We'd been using `prepublishOnly` prior to switching to Yarn Berry, so I am curious why we are choosing to use `prepare` here.

Reply:
Mainly I was worried about introducing something that people who update these packages aren't likely to be familiar with, but if we've already been using this pattern then that's fine.